src/Nordea/Components/Status.elm

```diff
@@ -85,3 +132,8 @@ blue text =
 gray : String -> List (Attribute msg) -> Html msg
 gray text =
     view text LightGray
+
+
+twoColors : String -> Color -> Color -> Float -> List (Attribute msg) -> Html msg
+twoColors text colorDone colorUndone pctDone =
```

Comment:
What do you think about making the `alpha 0.4` of the color an internal detail?
```suggestion
twoColors : String -> Color -> Float -> List (Attribute msg) -> Html msg
twoColors text colorDone pctDone =
```

Reply:
Did we btw abandon the convention of stacking calls like `withTitle` `withProgressBar` with these components? Here `withPartial Float` makes sense on top of already inited bar with color.

Reply:
I think the premiss of the `withPartiallyFilled Float` is a great option indeed! But I am a bit more fan (i think :P)  of having such params as a list instead of the builder pattern, e.g., like Coachmark